### PR TITLE
(#4535) - 5.1.0 release

### PR DESCRIPTION
**Pull request by daleharvey**
[skip ci]

### CODE REVIEW

File: docs/_posts/2015-11-03-cover-all.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+
+title: PouchDB 5.1.0 - Cover all the things
+author: Dale Harvey
+
+---
+
+It is time for the regularly scheduled PouchDB release, this month has a nice set of improvements and we even managed to remove a little(`3KB`) around the edges.
```
Comment (nolanlawson) [inline docs/_posts/2015-11-03-cover-all.md]: space between little and (

File: docs/_posts/2015-11-03-cover-all.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+### Coverage
+
+We are now very happy to be hitting [100% coverage on our tests](https://coveralls.io/github/pouchdb/pouchdb/). This currently only covers Node.js and excludes some difficult to test cases but it does give us confidence that our shared code is very well tested and we can ensure new code that is merged has the same level of testing.
```
Comment (nolanlawson) [inline docs/_posts/2015-11-03-cover-all.md]: diificult-to-test

Comment (daleharvey) [inline docs/_posts/2015-11-03-cover-all.md]: "cases that are difficult to test"

reads a little less awkward yeh?

File: docs/_posts/2015-11-03-cover-all.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+### GreenKeeper
+
+We have had issues in the past with how to manage npm dependencies. Version ranges can lead to other people breaking PouchDB with their releases and strict versions lead to us using stagnant packages and not being aware of upcoming breakages. During this month the [hood.ie team](http://hood.ie) released [GreenKeeper](http://greenkeeper.io) which gives us best of both worlds, stable versions that are always kept up to date!
```
Comment (nolanlawson) [inline docs/_posts/2015-11-03-cover-all.md]: Comma before "and strict" (coordinating subclauses) and after "During this month" (prepositional phrase).

File: docs/_posts/2015-11-03-cover-all.md (hunk 2 above)
Comment (nolanlawson) [inline docs/_posts/2015-11-03-cover-all.md]: Yup, I agree! :)

File: docs/_posts/2015-11-03-cover-all.md (hunk 1 above)
Comment (nolanlawson) [inline docs/_posts/2015-11-03-cover-all.md]: Comma after improvements too (subclause)

**Comment by nolanlawson** [comment]
I always nitpick on commas (they're to me what semicolons in JavaScript are for other people), but other than that, this is great! Ship it

**Pull request closed without merging**